cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -26,6 +27,17 @@ New features get added to the language continuously, and occasionally, some feat
 The following tables list all of the features which have been removed, deprecated, added, or extended in Cypher.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-4.1.3]]
+=== Version 4.1.3
+[options="header"]
+|===
+| Feature     | Type | Change | Details
+| `CREATE INDEX [name] IF NOT EXISTS FOR ...` | Syntax | Added | Makes index creation idempotent. If an index with the name or schema already exists no error will be thrown
+| `DROP INDEX name IF EXISTS` | Syntax | Added | Makes index deletion idempotent. If no index with the name exists no error will be thrown
+| `CREATE CONSTRAINT [name] IF NOT EXISTS ON ...` | Syntax | Added | Makes constraint creation idempotent. If a constraint with the name or type and schema already exists no error will be thrown
+| `DROP CONSTRAINT name IF EXISTS` | Syntax | Added | Makes constraint deletion idempotent. If no constraint with the name exists no error will be thrown
```

Comment:
the use of `name` and `[name]` is inconsistent - is that done intentionally?

Reply:
For `CREATE` the name is optional (`[name]`) while `DROP` is obligatory (`name`). Have you seen inconsistency between the `CREATE` commands?

Reply:
no, that's fine - it just stood out to me here. I wasn't aware the square brackets meant _optional_, and no brackets meant _required_.